testutil/integration/evmos/utils/bank.go

```diff
@@ -33,3 +36,29 @@ func CheckBalances(handler grpc.Handler, balances []banktypes.Balance) error {
 
 	return nil
 }
+
+// FundAccount is a helper function that funds a new account using
+// the private key of an existing account with funds
+func FundAccount(tf factory.TxFactory, funderPriv cryptotypes.PrivKey, addressToFund string, amount sdk.Coins) error {
```

Comment:
ditto

Reply:
yeah, we can upstream this one

Reply:
I believe that if we want to upstream a funding function we should use the bank Minter to not trigger other protocol logic like burning of Cosmos fee. These side behaviors can make the computation of expected results hard sometimes

Reply:
For the the current status on main, you can do that on an integration test, but not with the sdk v0.50 bump.
You have a couple of options, either you use a pre funded account (funded on genesis) or you fund the account you want to use with a funded accout. There're no shortcuts. The only way to change state is via a transaction included in a block.